src/main/kotlin/Calculator.kt

```diff
@@ -0,0 +1,48 @@
+import java.math.BigDecimal
+import java.math.MathContext
+import java.util.Stack
+
+class Calculator {
+    private fun add(i: BigDecimal, j: BigDecimal): BigDecimal =
+        i.add(j)
+
+    private fun subtract(i: BigDecimal, j: BigDecimal): BigDecimal =
+        i.subtract(j)
+
+    private fun multiply(i: BigDecimal, j: BigDecimal): BigDecimal =
+        i.multiply(j)
+
+    private fun divide(i: BigDecimal, j: BigDecimal): BigDecimal =
+        i.divide(j, MathContext.DECIMAL64)
+
+    fun calculate(strNum1: String?, strNum2: String?, operator: String): String {
+        val i = strNum1?.toBigDecimalOrNull() ?: throw IllegalArgumentException()
+        val j = strNum2?.toBigDecimalOrNull() ?: throw IllegalArgumentException()
+        return when (operator) {
+            Operator.ADD.symbol -> add(i, j)
+            Operator.MINUS.symbol -> subtract(i, j)
+            Operator.MULTIPLE.symbol -> multiply(i, j)
+            Operator.DIVIDE.symbol -> divide(i, j)
+            else -> throw IllegalArgumentException()
+        }.toString()
```

Comment:
`Operator`를 `String`타입으로 받고 있는데 `Operator` 타입을 이용해 enum 자체의 기능을 사용해보도록 해보는건 어떨까요?
```suggestion
        return when (operator) {
            Operator.ADD -> add(i, j)
            Operator.MINUS -> subtract(i, j)
            Operator.MULTIPLE -> multiply(i, j)
            Operator.DIVIDE-> divide(i, j)
            else -> throw IllegalArgumentException()
        }.toString()
```